### ISSUE TITLE
Incentive Clarity

### ISSUE THREAD

**Issue by paulhaas** [CommonsBuild/IH-intervention#14]
As an early hatcher and supporter of the Commons I am increasingly confused about the terminology used and various incentive schemes and interaction layers. I think it would help to clarify token structures, ownership and value.

We may be overengineering for a legal reality that no one may ever care about,

And under engineer for humans that care and want to help and understand their value.

**Comment by GriffGreen** [comment]
hahaha I don't know if people want to vote for this... 

I will close this issue tomorrow after people had time to read the response:

The TL;DR Context

1. Everyone who gets Praise (about 450 people) earns Impact Hours. 
2. Impact hours = GOVERNANCE! IH holders get 10-25% of the TEC Hatch tokens depending on how much is collected in the Hatch.
3. Impact hours = MONEY! The value of 1 Impact hour will be ~27-160 wxDai depending on how much is collected in the Hatch.

Each point explained

1. It's simple... Praise gets quantified into Impact Hours. 
[Here is the crazy spreadsheet ](https://docs.google.com/spreadsheets/d/1qvmwwlUHnQWYc2JQRoE1Qo_IKx6a4CI9ehWIMvkiqFc/edit#gid=1582481479) where every praise was assigned a number of Impact Hours. In the first tab you can see the complete list of Impact Hour holders. 


2. The exact % of governance power given to the people with Impact Hours is a curve dependent on much we raise: 
If we raise 800k, IH holders get 25% of the TEC Hatch tokens
If we raise 3mil, IH holders get 20% of the TEC Hatch tokens
If we raise 14m, IH holders get 10% of the TEC Hatch tokens

**The more we raise, the more relative governance is given to the people that put money in** #AlignedIncentives

3. The exact value of an Impact Hour is best calculated by relating it to how much wxDai it would take to mint the same amount of TEC Hatch Tokens. This is based off of an assumption of about 10,000 IH, if we have more than that then each IH will be worth less. If we have less than that then each IH will be worth more. 

Assuming no intervention, because some impact hours will not be claimed (only legal members of the Trusted Seed on the day of the Hatch will be able to recieve IH) This is a good estimation for the value of an impact hour. If the number of Impact Hours are increased by the winning praise 

If we raise 800k, 1 IH = 27.41 wxDai
If we raise 3mil, 1 IH = 77.07 wxDai
If we raise 14m, 1 IH = 159.82 wxDai

**The more we raise, the more each IH is worth** #AlignedIncentives

Below is a table with a bit more detail about 2 & 3, one important note is that TEC Hatch Tokens can be "Ragequit" and wxDai can be pulled out if people don't like the a vote, the direction of the TEC, or just want some money out. (TEC Hatch Tokens are non-transferable so will have no market value) The Ragequit value of each Impact Hour is actually the Ragequit % multiplied by Impact Hour Rate.

## Simulated Hatch Outcomes
|              | #Total wxDai Collected (wxDai) | Total Supply held by Builders (%) | Impact Hour Rate (wxDai) | Backer's RageQuit (%) |
|--------------|--------------------------------|-----------------------------------|--------------------------|-----------------------|
| Minimum Goal |                         $800,000 |                                25 |                    27.41 |                  73.5 |
|              |                        $1,000,000 |                             24.45 |                    33.25 |                 74.04 |
| Target Goal  |                        $3,000,000 |                                20 |                    77.07 |                  78.4 |
|              |                        $5,000,000 |                             16.92 |                   104.66 |                 81.42 |
|              |                        $7,000,000 |                             14.66 |                   123.62 |                 83.63 |
| Maximum Goal |                       $14,000,000 |                                10 |                   159.82 |                  88.2 |


I hope that clears up the terminology used and various incentive schemes and interaction layers and clarifies the token structures, ownership and value.

For a complete understanding of the Hatch DAO Parameters see the summary of the chosen Parameter set:
https://github.com/CommonsBuild/TEC-Hatch-Vote/issues/83

**Comment by papa-raw** [comment]
This was an extremely helpful answer and proposal that helped Curve Labs vote.

**Comment by paulhaas** [comment]
Amazing response, Griff! This was the only reason I made this post - talk
about the core econ more openly and directly. Big thanks and please
feel free to close it (but save your comment?)
*Paul Kohlhaas*
Founder
+ 1 845 248 27 83
Basel | Zurich | Berlin
www.linumlabs.com
  <https://twitter.com/LinumLabs>
<https://www.linkedin.com/company/linum-labs/?originalSubdomain=za>
<https://www.instagram.com/linumlabs/>
<https://www.facebook.com/LinumLabs/> <https://medium.com/linum-labs>

**Comment by jessicazartler** [comment]
Thank you for your questions and comments @paulhaas @papa-raw 🙏 Thank you @GriffGreen for your response. Would be great to move (copy/paste) this discussion to the forum before closing this https://forum.tecommons.org/t/pre-hatch-impact-hours-distribution-analysis/376/2 -- Griff having trouble copying/pasting yours with the chart, so if you have a chance before you close the issue to move this to forum, this is a great thread -thanks!

**Comment by jessicazartler** [comment]
@papa-raw curious to hear - what was the Curve labs vote? Look forward to seeing you at next research office hours - happy to share some things happening in gov research realm that could be interesting for you 🙏

**Comment by papa-raw** [comment]
Our votes went to "Praisemageddon" @jessicazartler

**Comment by jessicazartler** [comment]
Ohhhh I didn't mean that actually haha I interpreted as you had an unrelated Curve Labs vote / governance internal thing that this process supported somehow (my brain is funny)... I wouldn't ask how people voted normally but thanks for sharing and for commenting and your feedback! Hope to improve communications in the future around governance topics, glad Griff's breakdown was helpful. 🙏

**Comment by papa-raw** [comment]
my brain is fucked too these days, no worries

*Cheers,*

*/pat*

**Comment by liviade1** [comment]
I'm closing this issue, if you voted for it your votes will come back to your wallet and you can use it in other proposals, thanks for all the comments  🙏.
